support-frontend/assets/pages/contributions-landing/components/ContributionFormFields.jsx

```diff
@@ -61,7 +61,7 @@ const mapStateToProps = (state: State) => ({
   lastName: getCheckoutFormValue(state.page.form.formData.lastName, state.page.user.lastName),
   email: getCheckoutFormValue(state.page.form.formData.email, state.page.user.email),
   checkoutFormHasBeenSubmitted: state.page.form.formData.checkoutFormHasBeenSubmitted,
-  state: state.page.form.formData.state,
+  state: getCheckoutFormValue(state.page.form.formData.state, state.page.user.stateField),
```

Comment:
Not sure if this is needed or not, but it makes sense to be in parity with the other user-based fields.

Reply:
Just tested some more in CODE, and this should be removed as the stateField will change depending on what options the user chooses, but none of the other user fields will.